weave/legacy/tests/test_weave_api.py

```diff
@@ -34,23 +34,23 @@ def test_create_list_rename_delete():
     assert len(arts) == 0
 
 
-def test_weave_finish_unsets_client(client):
-    @weave.op
-    def foo():
-        return 1
-
-    weave.trace.client_context.weave_client.set_weave_client_global(None)
-    weave.trace.weave_init._current_inited_client = (
-        weave.trace.weave_init.InitializedClient(client)
-    )
-    weave_client = weave.trace.weave_init._current_inited_client.client
-    assert weave.trace.weave_init._current_inited_client is not None
-
-    foo()
-    assert len(list(weave_client.get_calls())) == 1
-
-    weave.finish()
-
-    foo()
-    assert len(list(weave_client.get_calls())) == 1
-    assert weave.trace.weave_init._current_inited_client is None
+# def test_weave_finish_unsets_client(client):
```

Comment:
this is a proper `trace` test

Reply:
oh, looks like you did! maybe delete then?

Reply:
done